### ISSUE TITLE
Which loss should I look at to check overfitting?

### ISSUE THREAD

**Issue by shenkev** [chenyuntc/simple-faster-rcnn-pytorch#60]
What metric should I check when training your model to see if it's overfitting?
